root/static/scripts/edit/URLCleanup.js

```diff
@@ -2023,12 +2023,15 @@ const CLEANUPS = {
     match: [new RegExp('^(https?://)?(www\\.)?operabase\\.com', 'i')],
     type: LINK_TYPES.otherdatabases,
     clean: function (url) {
-      return url.replace(/^(?:https?:\/\/)?(?:www\.)?operabase\.com\/a\/([^\/?#]+)\/([0-9]+).*$/, 'https://operabase.com/a/$1/$2');
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?operabase\.com\//, 'https://operabase.com/');
+      url = url.replace(/^https:\/\/operabase\.com\/a\/[^\/?#]+\/([0-9]+).*$/, 'https://operabase.com/artists/$1');
```

Comment:
Still support the older format in case users find them online, and to make it easier to update the existing ones.